### PR TITLE
sol

### PR DESCRIPTION
**Pull request by vladyslavchichkan**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,44 +1,44 @@
...
-        Message(toilet_id, MessageType.FLUSH),
-        Message(toilet_id, MessageType.CLEAN),
-    ]
-
-    # run the programs
-    service.run_program(wake_up_program)
-    service.run_program(sleep_program)
+    hue_light_id = await service.register_device(hue_light)
+    speaker_id = await service.register_device(speaker)
+    toilet_id = await service.register_device(toilet)
```
Comment (maxgerman) [inline app/main.py]: how can we register all of them concurrently?

**Review by maxgerman — CHANGES_REQUESTED** [review]
Good. See the comment

**Comment by Unlie9** [comment]
Image 22 08 2024 at 23 17
3 sec

**Review by maxgerman — APPROVED** [review]
well done

**Pull request closed without merging**